carbon-design-system/carbon
# fix(Tooltip): recalculate position upon open by keyboard

Fixes #3682.

{{short description}}

#### Changelog

**Changed**

- Add code to recalculate tooltip position when tooltip gets open by space/enter keys.

#### Testing / Reviewing

Testing should make sure `<Tooltip>` is not broken.

The fix can be verified by:

1. Focus on tooltip by tab key
2. Close the tooltip by hitting space key
3. Resize browser window
4. Re-open the tooltip by hitting space key
5. See if the tooltip position is right
